examples/jsm/nodes/materials/Line2NodeMaterial.js

```diff
@@ -13,6 +13,7 @@ import { uv } from '../accessors/UVNode.js';
 import { materialLineScale, materialLineDashSize, materialLineGapSize, materialLineDashOffset, materialLineWidth } from '../accessors/LineMaterialNode.js';
 import { viewport } from '../display/ViewportNode.js';
 import { dashSize, gapSize } from '../core/PropertyNode.js';
+import { color } from 'three/nodes';
```

Comment:
Inside of the Nodes system you cannot import from the export file (it would create a circular reference) -- you should import `color` from `../shadernode/ShaderNode.js`.